2996_reflection/reflection.md

```diff
@@ -3067,6 +3078,29 @@ Since namespace aliases are now entities, but their declarations are not definit
 
 ### [basic.def.odr]{.sref} One-definition rule {-}
 
+Add `$splice-expression$`s to the set of potential results of an expression in paragarph 3.
+
+::: std
+[3]{.pnum} An expression or conversion is _potentially evaluated_ unless it is an unevaluated operand ([expr.context]{.sref}), a subexpression thereof, or a conversion in an initialization or conversion sequence in such a context. The set of _potential results_ of an expression `$E$` is defined as follows:
```

Comment:
sref!

Reply:
Lmaooo I tried to be good, but I slipped up. Done.